jobs/v4/src/main/java/com/google/samples/JobSearchCreateCompany.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.samples;
+
+// [START job_search_create_company]
+
+import com.google.cloud.talent.v4beta1.Company;
+import com.google.cloud.talent.v4beta1.CompanyServiceClient;
+import com.google.cloud.talent.v4beta1.CreateCompanyRequest;
+import com.google.cloud.talent.v4beta1.TenantName;
+import com.google.cloud.talent.v4beta1.TenantOrProjectName;
+
+
+public class JobSearchCreateCompany {
+
+  public static void createCompany() {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "your-project-id";
+    String tenantId = "your-tenant-id";
+    String displayName = "your-company-display-name";
+    String externalId = "your-external-id";
+    createCompany(projectId, tenantId, displayName, externalId);
+  }
+
+  // Create a company.
+  public static void createCompany(
+      String projectId, String tenantId, String displayName, String externalId) {
+    // [START job_search_create_company_core]
+    try (CompanyServiceClient companyServiceClient = CompanyServiceClient.create()) {
+      // projectId = "Your Google Cloud Project ID";
+      // tenantId = "Your Tenant ID (using tenancy is optional)";
+      // displayName = "My Company Name";
+      // externalId = "Identifier of this company in my system";
+      TenantOrProjectName parent = TenantName.of(projectId, tenantId);
+      Company company =
+          Company.newBuilder().setDisplayName(displayName).setExternalId(externalId).build();
+      CreateCompanyRequest request =
+          CreateCompanyRequest.newBuilder()
+              .setParent(parent.toString())
+              .setCompany(company)
+              .build();
+      Company response = companyServiceClient.createCompany(request);
```

Comment:
Add a space above and below the request.

Code blocks can sorta follow the flow of:
1. Build the request
2. Make the request
3. Handle the response